src/components/PrivateRoute.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Redirect, Route } from 'react-router-dom'
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  return (
+    <Route
+      {...rest}
+      render={props =>
+        window.localStorage.getItem('authToken') ? (
```

Comment:
It'll be better to use isLoggedIn here

Reply:
I tried, there is something weird with scope and any function outside is unavailable

Reply:
Ok, let leave it as is for the moment